package.json

```diff
@@ -25,6 +25,6 @@
     "nyc": "^14.1.1"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
When rebasing, note that this is no longer necessary after e57c0725d1f5df559b78bdb50bb930af8f05d93b.